bin/start-shuffle-server.sh

```diff
@@ -67,7 +67,7 @@ JAVA_LIB_PATH="-Djava.library.path=$HADOOP_HOME/lib/native"
 echo "class path is $CLASSPATH"
 
 MAX_DIRECT_MEMORY_OPTS=""
-if [ -n "$MAX_DIRECT_MEMORY_SIZE" ]; then
+if [ -n "${MAX_DIRECT_MEMORY_SIZE:-}" ]; then
```

Comment:
Why don't "RSS_IP" have similar problem?

Reply:
Let me take a look. Do you test when deploying?

Reply:
Due to `set +o nounset` ?

Reply:
Yes, we should keep consistent style with other places.

Reply:
So let us modify other places?

Reply:
Yes. Let me update tomorrow

Reply:
Done